test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -1908,7 +1912,7 @@ def check_multi_bldg_handling(baseline_base)
         wwr_baseline = run_query_tabulardatawithstrings(model_baseline, 'InputVerificationandResultsSummary', 'Conditioned Window-Wall Ratio', 'Gross Window-Wall Ratio', 'Total', '%').to_f
         # Check WWR against expected WWR
         wwr_goal = 100 * @@wwr_values[building_type].to_f
-        assert(wwr_baseline < wwr_goal, "Baseline WWR for the #{building_type}, #{template}, #{climate_zone} model with user data is incorrect. The WWR of the baseline model is #{wwr_baseline} but should be #{wwr_baseline}, smaller than the WWR goal #{wwr_goal}")
+        assert(wwr_baseline > wwr_goal, "Baseline WWR for the #{building_type}, #{template}, #{climate_zone} model with user data is incorrect. The WWR of the baseline model is #{wwr_baseline} but should be greater than the WWR goal #{wwr_goal}")
```

Comment:
Shouldn't we be looking for a difference instead? As in is `wwr_baseline` different than `wwr_goal`.

Reply:
This was a bug that motivated me to work on the unit test...

It is actually the multi-bldg-handling check function. Background story is, this test case is a small office with three rooms assigned to three different types of WWR space types. So in total, there are four different WWR space types.
- Retail (stand alone) - 11%  - Perimeter_ZN_2
- Retail (strip mall) - 20% -  - Perimeter_ZN_1
- Restaurant (quick service) - 34% - Perimeter_ZN_4
- Office (<=5000 ft2) - 19%.
The `wwr_goal` is 19% because it is from the json file.

Previously, the WWR function can only shrink, so obviously, because of the Retail (stand alone), the total WWR should be smaller than 19%.

With the new code, Restaurant and Retail strip mall both have higher WWR and the total WWR is slightly over 20% so the generated `wwr_baseline` should be higher (>) than `wwr_goal`.

To be more precise, I can use (wwr_baseline - wwr_goal) == 1% but so far > operator might be OK. Let me know your thoughts.

Reply:
That sounds good!